<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeedsDeepDialogBase</class>
 <widget class="QDialog" name="WeedsDeepDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QGIS - WeedsDeep</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>521</width>
     <height>261</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>501</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="body">
     <item row="1" column="0">
      <widget class="QLabel" name="tiffdat_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;TIFF.dat&lt;/span&gt; path:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="shpdat_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;SHP.dat&lt;/span&gt; folder:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QgsFileWidget" name="rawdat_browser">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="filter">
        <string>Text File (*.dat)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QgsFileWidget" name="shpdat_browser">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="filter">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="rawdat_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;RAW.dat&lt;/span&gt; path:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QgsFileWidget" name="tiffdat_browser">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="filter">
        <string>Text File (*.dat)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="shpfolder_label">
       <property name="text">
        <string>Shapefiles folder:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QgsFileWidget" name="shpfolder_browser">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;This plugin generates shapefiles from given &lt;/span&gt;&lt;span style=&quot; font-size:7pt; font-weight:600;&quot;&gt;RAW.dat&lt;/span&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt; and &lt;/span&gt;&lt;span style=&quot; font-size:7pt; font-weight:600;&quot;&gt;TIFF.dat&lt;/span&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;:&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-size:7pt; font-weight:600;&quot;&gt;SHP.dat folder&lt;/span&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt; - folder where to generate &lt;/span&gt;&lt;span style=&quot; font-size:7pt; font-weight:600;&quot;&gt;SHP.dat&lt;/span&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt; file, it contains paths to &lt;/span&gt;&lt;span style=&quot; font-size:7pt; font-style:italic;&quot;&gt;.shp&lt;/span&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt; files.&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-size:7pt; font-weight:600;&quot;&gt;Shapefiles folder &lt;/span&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;- folder where to save the generated &lt;/span&gt;&lt;span style=&quot; font-size:7pt; font-style:italic;&quot;&gt;.shp&lt;/span&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt; files. (subfolders are created)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="header">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>QGIS - WeedsDeep</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>WeedsDeepDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>WeedsDeepDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
